recipes/doc2dash/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Hynek Schlawack
```

Comment:
You don't need this extra license file since it is included in the source distribution.

Reply:
Removed license file in https://github.com/conda-forge/staged-recipes/pull/4121/commits/e2d466fc2af5b859c9318cc6df2c9d8143e35d9d